doc/api/inspector.md

```diff
@@ -11,94 +11,209 @@ inspector.
 
 It can be accessed using:
 
-```js
+```mjs
```

Comment:
Doc-wise, this should be somewhere else, And marked with `> Stability 1 Experimental.

E.g.:

````
## Promises

<!-- introduced_in=v19.0.0 -->

> Stability: 1 - Experimental

<!-- source_link=lib/inspector/promises.js

[...] Description.
```

IMHO this is a doc-blocker.

Reply:
It is marked as experimental, see https://github.com/nodejs/node/pull/44250/files#diff-d6da7e100224e744520fde97ca99f743ad9eafc83d50ed92ed2b7870f0589f68R34

`<!-- introduced_in=v19.0.0 -->` would not be appropriate here, those are to reference the first version that included the markdown document in the docs. This PR does contain a YAML comment block that does the same thing: https://github.com/nodejs/node/pull/44250/files#diff-d6da7e100224e744520fde97ca99f743ad9eafc83d50ed92ed2b7870f0589f68R37.

I disagree this is a doc blocker, also doc can always be fixed in a semver patch, while the module can only land in a semver-major.

Reply:
Not saying this is a PR blocker, but felt for me like a doc blocker, if you disagree that's fine, I don't want to block this PR too much 👀

Reply:
I didn't get whether i should change it or not haha. Is it ready to be landed? cc @aduh95 @ovflowd

Reply:
Perfect! @aduh95. 

for the next PR @ovflowd I didn't get what you suggested to change. I plan to:

change connect and disconnect functions to be async (this might lead me to change something in cpp to get those events (no idea how to do it haha)) and what you suggested adding here.

Reply:
Hmm, @ErickWendel not sure, but I was just making suggestions to the API docs you were writing xD

But it's all fine and this landed 🎉